eb

```diff
@@ -13,7 +13,11 @@ if [[ $HOSTNAME =~ archimedes.c3.ca$ ]]; then
 	export PATH=$PATH:/opt/software/slurm/bin
 	export EASYBUILD_JOB_BACKEND='Slurm'
 	export SBATCH_MEM_PER_CPU='3500m'
+	if [[ $CURRENT_USER == "ebuser" ]]; then
+		export SBATCH_OUTPUT="/cvmfs/local/var/log/slurm-%x-%j.out"
+	fi
```

Comment:
I'm afraid we either need to change framework or look into g3cpie? The framework sets `--output=<name>-%j.out` explicitly.

Reply:
Yeah, that's what I conclude too with Kenneth's answer on EB slack